### PR TITLE
[#169785262] Fix profile create and update

### PR DESCRIPTION
**Pull request by francescopersico**

### CODE REVIEW

File: utils/profiles.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ export function retrievedProfileToExtendedProfile(
     accepted_tos_version: profile.acceptedTosVersion,
     blocked_inbox_or_channels: profile.blockedInboxOrChannels,
     email: profile.email,
-    is_email_enabled: profile.isEmailEnabled === true,
-    is_email_validated: profile.isEmailValidated === true,
+    is_email_enabled: profile.isEmailEnabled !== false,
```
Comment (gunzip) [inline utils/profiles.ts]: why this change ?

Comment (francescopersico) [reply utils/profiles.ts]: This property is `undefined` in old user profile (created before `isEmailEnabled` was available), but we still want to return an ExtendedProfile that has `is_email_enabled` set to `true` in this case.

Comment (francescopersico) [reply utils/profiles.ts]: Same for is_email_validated.
